Off-Top-App/off-top-java
# OT-34 | removed stomp websocket, am now using a less intense websocket, also …

**What I Did:** Removed stomp WebSocket, am now using a less intense WebSocket, also created a service that will handle incoming data from the clients WebSocket

**How To Test:**
1. Make sure you have flutter running on client. For instructions go this link: https://github.com/Off-Top-App/off-top-flutter/compare/OT-34?expand=1
2. Then run the spring-boot server. I like to go to off-top-java in my terminal and then run `mvn spring-boot:run` however I'm not sure if Windows users get direct access to `mvn` 
3. In the mobile app, click on test websocket connection and you should see the spring boot server updating like in the photo below: 
<img width="654" alt="Screen Shot 2020-02-25 at 5 36 48 PM" src="https://user-images.githubusercontent.com/26909882/75303494-bf852700-57f5-11ea-8b9f-67de455d56d9.png">
